config/config.ini

```diff
@@ -20,7 +20,23 @@
 [Discord]
 Key = KEY
 
+[Server]
```

Comment:
remove the names and IDs specific to the discord server. if you want to generalize things you need to be consistent. and wasn't this file supposed to be ignored?

Reply:
It’s still generalized if specific stuff is relegated to config and can be changed easily. Be careful about removing since then the auto deploy will break unless you guys have implemented a fix for that

Reply:
(to clarify: i think it's best if this stuff isn't hardcoded in config, although it doesn't matter too much, but you'll have to make sure the deployment system is good [and also generalizable lol] first)

Reply:
Yeah, I forgot but I wanted to change the readme too, right now it says go to config.ini to change the token but now it should say that there are other values to change. And yeah this is already better than what we had before, where you had to go to each file individually to and change every direct mention of the server ID, roles etc, but if you guys have a better way go for it